tests/test_crash_reports.py

```diff
@@ -107,6 +107,34 @@ def test_that_top_crashers_reports_links_work(self, mozwebqa, product):
             top_crashers = csp.release_channels
 
     @pytest.mark.nondestructive
+    def test_top_crasher_reports_tab_has_uuid_report(self, mozwebqa):
+        csp = CrashStatsHomePage(mozwebqa)
+        top_crashers = csp.click_last_product_top_crashers_link()
+        crash_signature = top_crashers.click_first_signature()
+        crash_signature.click_reports_tab()
+        reports_table_count = len(crash_signature.reports)
+
+        # verify crash reports table is populated
+        Assert.greater(crash_signature.results_count_total, 0)
+        Assert.greater(reports_table_count, 0, "No reports found")
+
+        most_recent_report = crash_signature.reports[0]
+        uuid_report = most_recent_report.click_report_date()
+
+        # verify the uuid report page
+        Assert.equal(uuid_report.uuid_in_body, uuid_report.uuid_in_table,
+                     'UUID in body did not match the UUID in the table: '
+                     'body "%s", table "%s"'
+                     % (uuid_report.uuid_in_body,
+                        uuid_report.uuid_in_table))
+        Assert.equal(uuid_report.signature_in_body,
+                     uuid_report.signature_in_table,
+                     'Signature in body did not match the signature in the '
+                     'table: body "%s", table "%s"'
+                     % (uuid_report.signature_in_body,
+                        uuid_report.signature_in_table))
```

Comment:
These are just asserting that the value is the same in the header and table, which would pass even if the value was an empty string. If such as check is covered elsewhere then perhaps this is not a concern.